tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -196,6 +248,35 @@ private void ConfigureTraceTransport(IConfigurationSource source, bool isWindows
                     ?? 500;
 #endif
             }
+            else
+            {
+                // Check for UDS
+                if (isConventionBasedSocket)
```

Comment:
can never be true right? We set it only in the first if that is followed by elses.

Reply:
Good catch, I need to write some tests for ExporterSettings on this note.